### DISCUSSION TITLE
Octernship Application

### DISCUSSION THREAD

**Discussion by shivansh-13** [community/community#58357] [GitHub Education]
### Hi everyone,

Hi everyone!!
So I am interested in DiveHQ Calorie API. I clicked on the button " I am interested" I showed around 480/500 students applied. And then, I was given a task to complete. **I am almost done with my task, but now it says 500/500 applied.** 

**Does clicking on "I am interested" count as applied, or only those who complete the task are considered as applied??** 

**Since it now shows 500/500  applied, should I submit my task?** Am I eligible for octernship or not?
[EDIT: the deadline for submission is 19th June]
Thanks and Regards.

**Comment by saaxena** [comment]
Yes you are completely eligible. 500 is the no of their intake capacity of students.
